pages/blog/february-2021-at-asyncapi.md

```diff
@@ -0,0 +1,184 @@
+---
+title: February 2021 at AsyncAPI
+date: 2021-03-16T06:00:00+01:00
+type: Communication
+tags:
+  - status
+cover: /img/posts/february-2021-at-asyncapi-cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+excerpt: We are getting close to joining a foundation. We started a few new initiatives. We are busy. In other words, this is a good time to join us and drive things together.
+featured: true
+---
+
+> Read [January 2021 at AsyncAPI](/blog/january-2021-at-asyncapi) for the update from January.
+
+I enjoy monthly status over weekly one. It is not that I'm just happy I have more time for other things. Content size did not change. I think people do have other things to do than following AsyncAPI only :sweat_smile: Life in open-source runs slower, at least when you look at it from the outside. I have an impression, that more people pay attention to the status when it is once a month. I wonder what your view is on that.
+
+## Open governance model aka charter ready for the review
+
+Is an "open governance model" the same as a "charter"? No. Charter is a boring legal document that describes many rules that need to be followed when being under a foundation. One of the things the charter tackles is the general rules on how the project will operate and be governed. 
+
+I think the most important thing to write here is that the AsyncAPI charter is a reality. It finally arrived and is ready for review. In the special blog post, we also explained what governance model we envision for the AsyncAPI Initiative in more human-friendly words. This content was released and communicated on Tuesday, 9th of March, and we will wait for your feedback until the 23rd of March. It looks like we will kick off April under the umbrella of the foundation :crossed_fingers:
+
+Please take your time to:
+- Read [Finding a Good Open Governance Model for AsyncAPI](/blog/governance-motivation) to understand our motivation
+- Check out [this](https://github.com/asyncapi/.github/pull/37) pull request with the charter
+
+## Code generators activities
+
+We had some significant traffic in the area of code generation templates in February.
+
+### PHP
+
+[Emiliano Zublena](https://twitter.com/emilianozublena) joined the AsyncAPI community big time by starting with donating [a new template for PHP](https://github.com/asyncapi/asyncapi-php-template). It is not yet released under **@asyncapi** scope on npm, but you can already play with it by using the AsyncAPI Generator with a direct GitHub link like:
```

Comment:
I'm not active at all on Twitter, it might be a better idea to link either my github acc or linkedin acc :)

Reply:
@emilianozublena  please open PR with the fix and I merge